cleanlab/classification.py

```diff
@@ -292,7 +293,7 @@ def fit(
           The classifier that this instance was initialized with,
           ``clf``, must be able to fit() and predict() data with this format.
 
-        labels : np.array or pd.Series
+        y : np.array or pd.Series
           An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
           Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
```

Comment:
> Should something like this be added to the docstring? What SemVer should be here?

"""
.. deprecated:: 2.1.0
          `labels` will be removed in cleanlab 3.0.0. Use `y` instead to provide the labels.
"""

Instead of that, let's just add the following extra statement at the end of the docstring for argument `y`:

Note: this argument was previously called ``labels`` in some previous versions of cleanlab.